src/udev/udev-builtin-path_id.c

```diff
@@ -397,7 +397,7 @@ static struct udev_device *handle_scsi_default(struct udev_device *parent, char
         return hostdev;
 }
 
-static struct udev_device *handle_scsi_hyperv(struct udev_device *parent, char **path) {
+static struct udev_device *handle_scsi_hyperv(struct udev_device *parent, char **path, unsigned int guid_str_len) {
```

Comment:
I think this should be `size_t`. Also, I note that we use `unsigned` instead of `unsigned int`.